doc/user/COLOR_MAPS.md

```diff
@@ -0,0 +1,41 @@
+# Color maps
+
+## Image files
+
+It is possible to specify a color map using the `--colormap-file` option.
+The value of the option can be a full path to an image file, a filename only is which case, these directories are used to find the file (in this order):
```

Comment:
The value of the option can be a path to an image file, a filename or a filestem.
If it is not a path, these directories are used to find the file (in this order):

Reply:
any image file ? which extensions are supported ?

Reply:
mention that the first file found will be used.

Reply:
done